plugins/modules/cpanm.py

```diff
@@ -142,7 +142,19 @@
     version: '1.0'
 """
 
+RETURN = """
+---
+cpanm_version:
+  description: Version of CPANMinus.
+  type: str
+  returned: always
+  sample: 1.7047
```

Comment:
Since this looks like a float, it's better to add quotes:
```suggestion
  sample: "1.7047"
```
For 1.7047 it's likely not a problem, but for 1.7040 it would be :)